KoTH/Standard/Wuhu Island/map.xml

```diff
@@ -0,0 +1,247 @@
+    <control-point name="Summer Castle" points="1"> <!-- Alternative name: "Castle" or "Summerstone Castle" -->
+        <capture>
+            <cylinder id="summer-castle" base="-1205.5,53,400.5" radius="4" height="4"/>
+        </capture>
+        <progress>
+            <region id="summer-castle"/>
+        </progress>
+        <captured>
+            <region id="summer-castle"/>
+        </captured>
+    </control-point>
+    <control-point name="Mysterious Ruins" points="1"> <!-- Alternative name: "Ruins" -->
+        <capture>
+            <cylinder id="mysterious-ruins" base="-1146.5,30,269.5" radius="4" height="4"/>
+        </capture>
+        <progress>
+            <region id="mysterious-ruins"/>
+        </progress>
+        <captured>
+            <region id="mysterious-ruins"/>
+        </captured>
+    </control-point>
+    <control-point name="Maka Wuhu" points="1"> <!-- Alternative name: "Volcano" -->
+        <capture>
+            <cylinder id="maka-wuhu" base="-1129.5,63,369.5" radius="4" height="4"/>
+        </capture>
+        <progress>
+            <region id="maka-wuhu"/>
+        </progress>
+        <captured>
+            <region id="maka-wuhu"/>
+        </captured>
+    </control-point>
+    <control-point name="Cedar Tree Cave" points="1"> <!-- Alternative name: "Hill" or "Cedar Tree Tunnel" -->
+        <capture>
+            <cylinder id="cedar-tree-cave" base="-1120.5,45,422.5" radius="4" height="4"/>
+        </capture>
+        <progress>
+            <region id="cedar-tree-cave"/>
+        </progress>
+        <captured>
+            <region id="cedar-tree-cave"/>
+        </captured>
+    </control-point>
+    <control-point name="Cocoba Hotel" points="1"> <!-- Alternative name: "Resort" or "Tennis Court" -->
+        <capture>
+            <cylinder id="cocoba-hotel" base="-1036.5,21,437.5" radius="4" height="4"/>
+        </capture>
+        <progress>
+            <region id="cocoba-hotel"/>
+        </progress>
+        <captured>
+            <region id="cocoba-hotel"/>
+        </captured>
+    </control-point>
+    <control-point name="19th Hole Hotel" points="1"> <!-- Alternative name: "Wedge" -->
+        <capture>
+            <cylinder id="nineteeth-hole-hotel" base="-1421.5,26,883.5" radius="4" height="4"/>
+        </capture>
+        <progress>
+            <region id="nineteeth-hole-hotel"/>
+        </progress>
+        <captured>
+            <region id="nineteeth-hole-hotel"/>
+        </captured>
+    </control-point>
+</control-points>
+<!-- Filters -->
+<filters>
+    <team id="yellow-only">yellow</team>
+    <team id="green-only">green</team>
+    <team id="blue-only">blue</team>
+    <team id="red-only">red</team>
+</filters>
+<!-- Regions -->
+<regions>
+    <!-- Definitions -->
+    <union id="spawn-protections">
+        <cuboid id="yellow-spawn-protection" min="-1150,30,369" max="-1158,36,377"/>
+        <cuboid id="green-spawn-protection" min="-1162,60,358" max="-1169,67,351"/>
```

Comment:
`<cuboid id="green-spawn-protection" min="-1159.5,60,351.5" max="-1170.5,67,359.5"/>`

- Green's spawn region, appeared to be the only region that needed a fix, it was off by one block, and players could enter the spawn. This should fix it.